content/cloud/user-settings.md

```diff
@@ -0,0 +1,73 @@
+- [Delete Your Fermyon Cloud Account](#delete-your-fermyon-cloud-account)
+- [Next Steps](#next-steps)
+
+## Fermyon Cloud User Settings
+
+User settings empower you to take control of your Fermyon Cloud experience. Today, you can complete the following actions on user setting: create and manage a Personal Access Token (PAT); delete your Fermyon Cloud account. Follow the relevant sections for your use case below to learn how to complete both of these actions. 
+
+## Create and Manage a Personal Access Token (PAT)
+
+### Create A PAT
+
+To deploy your Spin applications in a streamlined manner, you may want to create a PAT to use with a continuous integration pipeline such as GitHub Actions. Let's walk through the steps required to create a PAT.
+
+First [log into Fermyon Cloud](/quickstart.md#log-in-to-the-fermyon-cloud). Hover over your username in the top right corner and select the 'User Settings' from the down options:
+
+![User settings](/static/image/user-setting.png)
+
+Now select the "Add a token" button under the Personal Access Tokens heading. 
+
+![User settings view](/static/image/user-setting-view.png)
+
+Input your preferred token name. We recommend that you use a  descriptive name to help you can recall its purpose later. 
+
+![Create token view](/static/image/create-token.png)
+
+Take a moment to copy your token name and store it in a safe location for later. 
+
+![Create token confirmation](/static/image/create-token-confirmation.png)
+
+Exit out of the panel. Congratulations on creating your first PAT with Fermyon Cloud. 
+
+![User Settings with Fermyon Cloud](/static/image/user-settings-with-token.png)
+
+### Delete A PAT
+
+To delete a PAT, click on the `x` to the right of the desired token name. This will bring up the following prompt:
+
+![Delete token confirmation](/static/image/delete-token-confirmation.png)
+
+Type out the complete token name and hit delete to permanently delete your PAT. 
+
+![Delete token confirmation final warning](/static/image/delete-token-confirmation-2.png)
+
+You should no longer see the PAT in your User Settings view.
+
+![User settings view](/static/image/user-setting-view.png)
+
+## Delete Your Fermyon Cloud Account
+
+You may wish to delete your Fermyon Cloud account. This is a permanent action that will delete your account along with any running applications associated with the account at that time.
+
+First [log into the Cloud](/quickstart.md#log-in-to-the-fermyon-cloud). Hover on your username in the top right corner and select the 'User Settings' from the down options:
+
+![User settings](/static/image/user-setting.png)
+
+Click the "Delete account" button in the _Danger Zone_ box.
+
+![User settings view](/static/image/user-setting-view.png)
+
+Select all three check boxes and select "Delete account" proceed with account deletion. Note this is a permanent action and will result in the deletion of any running applications in Fermyon Cloud. 
```

Comment:
"result in the deletion of" -> "delete"?

Reply:
Reworded.